stdlib/public/TensorFlow/Ops.swift

```diff
@@ -589,7 +589,7 @@ public extension Tensor where Scalar : TensorFlowFloatingPoint {
   @differentiable(wrt: self)
   func standardDeviation() -> Tensor {
     // Reduce along all dimensions.
-    return standardDeviation(alongAxes: Array(0..<shape.rank))
+    return standardDeviation(alongAxes: Array(0..<Int32(shape.rank)))
```

Comment:
All axis-taking ops should take `Int` instead of `Int32`. It is better to incorporate those changes into this PR.